gitversion.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+_tag=`git describe`
+another_var=`expr index "$_tag" "-"`
```

Comment:
`another_var` is not a very expressive variable name. I guess this should be `suffix_index`? But see my following more general comment...